ras/ras_ppcutils.py

```diff
@@ -291,10 +291,22 @@ def test_ls_vdev(self):
         """
         self.log.info("===============Executing ls-vdev tool test===="
                       "===========")
+        self.is_fail = 0
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
-        self.error_check()
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
```

Comment:
@shirishaganta1 why cann't we capture this while we run the same command in line 295?

Reply:
run_cmd() may not get you the output to split.. this could be exit status check